src/EditorFeatures/Core.Wpf/InlineRename/AbstractEditorInlineRenameService.SymbolRenameInfo.cs

```diff
@@ -21,7 +24,7 @@ internal abstract partial class AbstractEditorInlineRenameService
         /// <summary>
         /// Represents information about the ability to rename a particular location.
         /// </summary>
-        private partial class SymbolInlineRenameInfo : IInlineRenameInfo
+        private partial class SymbolInlineRenameInfo : IIInlineRenameInfoWithFileRename
```

Comment:
1. `III` :)
2. will have to see what's going on later in the review, but need to make sure this won't be a problem downstream.